### DISCUSSION TITLE
When will <model-viewer> support new material PBR properties?

### DISCUSSION THREAD

**Discussion by Dmitrijsmok** [google/model-viewer#1869] [Q&A]
Hey guys,
Is it possible to make model-viewer to enable/support Sheen. 
This node is must-have for PBR fabric material
It is already supported for gLTF files and three.js:
https://threejs.org/examples/?q=sheen#webgl_materials_physical_sheen

**Comment by cdata** [comment]
In general we intend to support whatever Three.js supports regarding glTF. Three.js gains support for glTF PBR via updates to its `GLTFLoader`. Whatever the `GLTFLoader` produces,`<model-viewer>` will use.

Has sheen been specified and accepted for glTF? cc @donmccurdy

**Comment by donmccurdy** [comment]
Sheen is not specified in glTF yet, but it's coming.

Of the new material parameters planned for glTF, I think Clear Coat will be first.

**Comment by Dmitrijsmok** [comment]
Great, thank you, guys!
